sources/packages/backend/apps/api/src/route-controllers/cas-supplier/cas-supplier.aest.controller.ts

```diff
@@ -0,0 +1,88 @@
+import {
+  Body,
+  Controller,
+  Get,
+  Param,
+  ParseIntPipe,
+  Post,
+} from "@nestjs/common";
+import { ApiTags } from "@nestjs/swagger";
+import { AuthorizedParties, IUserToken, Role, UserGroups } from "../../auth";
+import {
+  AllowAuthorizedParty,
+  Groups,
+  Roles,
+  UserToken,
+} from "../../auth/decorators";
+import BaseController from "../BaseController";
+import {
+  AddCASSupplierAPIInDTO,
+  CASSupplierInfoAPIOutDTO,
+} from "./models/cas-supplier.dto";
+import { CASSupplierService } from "../../services";
+import { ClientTypeBaseRoute } from "../../types";
+
+/**
+ * CAS supplier controller.
+ */
+@AllowAuthorizedParty(AuthorizedParties.aest)
+@Groups(UserGroups.AESTUser)
+@Controller("cas-supplier")
+@ApiTags(`${ClientTypeBaseRoute.AEST}-cas-supplier`)
+export class CASSupplierAESTController extends BaseController {
+  constructor(private readonly casSupplierService: CASSupplierService) {
+    super();
+  }
+
+  /**
+   * Gets CAS suppliers for a given student.
+   * @param studentId student id to retrieve CAS suppliers.
+   * @return a list of CAS supplier info.
+   */
+  @Get("student/:studentId")
+  async getCASSuppliers(
+    @Param("studentId", ParseIntPipe) studentId: number,
+  ): Promise<CASSupplierInfoAPIOutDTO[]> {
```

Comment:
I believe the preferred way for the DTOs would be to have the array as a property.
I tried to find similar examples and found somewhere a DTO was created with a single array property.